test/config/index.js

```diff
@@ -37,7 +37,7 @@ const IBM_GITHUB_AUTH = process.env.GIT_EXPORT_AUTH || '';
 const K8S_NAME_SPACE = process.env.KUBE_NAMESPACE || 'default';
 
 const containerDir = '/home/cwuser/codewind-test-projects/';
-const dir = '/tmp/codewind-test-projects/';
+const TEMP_TEST_DIR = __dirname;
```

Comment:
I'm not sure about using `__dirname` here, that will resolve to the directory of this modue: https://nodejs.org/docs/latest/api/modules.html#modules_dirname
I'm not sure if that's a good temp dir. If the tests are run locally will we then have test data under the users source tree? That might not be .gitignored and could end up getting committed. (Or I may have missed something about how this is used!)